centaur/src/main/resources/standardTestCases/failing_return_code.test

```diff
@@ -0,0 +1,11 @@
+name: failing_return_code
+testFormat: workflowfailure
+
+files {
+  workflow: failing_return_code/failing_return_code.wdl
+}
+
+metadata {
+  workflowName: FailingReturnCode
+  status: Failed
+}
```

Comment:
This angry little circle that Github adds means there is no newline at the end of the file. We try to make sure all files end with newlines, it just makes them easier to work with. For example, your terminal doesn't get messed up when you `cat` them.